src/ui/main_window.ui

```diff
@@ -208,7 +209,7 @@
   </action>
   <action name="dactionIRC">
    <property name="text">
-    <string>IRC help channel</string>
+    <string>IRC</string>
```

Comment:
I wonder if it would be better to keep some description like "help" or "chat" around, because not every player may know immediately what "IRC" and "Discord" actually refer to